### PR TITLE
Implemented stub function for show interface

### LINKED ISSUE #57: Implement the stub function show traffic
Work for sprint1 - master issue is #20

### PR DESCRIPTION
**Pull request by m-maque96**
Pull request for issue #19

### CODE REVIEW

File: asa_parser/src/asa_parse.py [hunk 1]
```diff
@@ -63,6 +63,10 @@ def show_tech_support_detail(self):
         """Parser for show cpu detailed"""
         return json.dumps({'text': self.get_show_section('tech-support detail')})
 
-    def show_context_details(self):
-        """Parser for show context details"""
```
Comment (dmeyer3691) [inline asa_parser/src/asa_parse.py:67]: Why remove this? This was part of another teams sprint and should remain part of the codebase. Please add it back in and we can approve this pull request

File: demo.py [hunk 2]
```diff
@@ -42,9 +42,11 @@ def run_demo():
     # ipsec stats
     print(primary_asa.ipsec_stats())
 
-    # context details
-    print(primary_asa.show_context_details())
```
Comment (dmeyer3691) [inline demo.py:46]: Why remove this? This was part of another teams sprint and should remain part of the codebase. Please add it back in and we can approve this pull request

**Review by dmeyer3691 — CHANGES_REQUESTED** [review]
Please add back in the code for show_context_details and we can approve this pull request

File: demo.py (hunk 2 above)
Comment (m-maque96) [reply demo.py:46]: Oh, I never deleted that. I am not sure how that happened. I just added my stub function. @dmeyer3691

Comment (m-maque96) [reply demo.py:46]: I added back the show context details stub function @dmeyer3691

Comment (dmeyer3691) [reply demo.py:46]: Thanks, looks good now!

**Review by dmeyer3691 — APPROVED** [review]

**Pull request merged**